client/pages/BoardPage.jsx

```diff
@@ -48,6 +48,21 @@ function BoardPage({user}) {
       getBoard();
     }, [])
 
+    const navigate = useNavigate();
+
+    async function logout() {
+      try {
+        // make DB call to terminate session
+        const logoutResponse = await axios.delete('/logout');
```

Comment:
do we need `logoutResponse` for anything? If not, can we possibly just do:

```suggestion
        await axios.delete('/logout');
```

Reply:
Nope!  it was for checking the username object on the req.session and I never updated it.  Your change is good and has been implemented